CanvasColorSpaceProposal.md

```diff
@@ -3,11 +3,11 @@
 ## Use Case Description
 * Contents displayed through a canvas element should be color managed in order to minimize differences in appearance across browsers and display devices. Improving color fidelity matters a lot for artistic uses (e.g. photo and paint apps) and for e-commerce (product presentation).
 * Canvases should be able to take advantage of the full color gamut and dynamic range of the display device.
-* Some applications prefer that compositing, filtering and interpolation calculations be performed in a physically based linear color space.
```

Comment:
Linear-space blending was a goal of the original proposal, but I agree with your removal of it.